app/admin/inboxes/page.tsx

```diff
@@ -44,9 +44,22 @@ export default async function InboxesPage({
     const offset = (currentPage - 1) * pageSize;
 
     allInboxes = await db
```

Comment:
Something about this new query is causing the page to crash
<img width="986" height="175" alt="Screenshot 2025-09-14 at 4 58 49 PM" src="https://github.com/user-attachments/assets/1c1ca545-921c-4b2c-946f-77d875bf77e8" />

I would expect this query to look similar to what we're doing for contacts